src/lio/os/file.c

```diff
@@ -1128,7 +1128,7 @@ int va_timestamp_set_attr(lio_os_virtual_attr_t *va, lio_object_service_fn_t *os
     n = (int)(long)va->priv;  //** HACKERY ** to get the attribute prefix length
     key = &(fullkey[n+1]);
 
-    if (strlen(fullkey) < n) {  //** Nothing to do so return;
+    if ((int)strlen(fullkey) < n) {  //** Nothing to do so return;
```

Comment:
That's true. But blindly casting away the error is not always correct,
hence my question. Signedness has strange spec-defined behavior that you
could be papering over.



-- 
It's dark in this basement.